day_7/10_little_monkeys_with_hash.rb

```diff
@@ -0,0 +1,127 @@
+require 'humanize'
+
+# Version 2
+
+monkey_jump = {1 => "One", 2 => "Two", 3 => "Three"}
+
+print "How many monkeys are jumping on the bed?: "
+x = gets.chomp.to_i
+until x > 0
+  puts "Input a whole number greater than zero"
+  x = gets.chomp.to_i
+end
+
+if x == 1
```

Comment:
I would schedule some time with your mentor to refactor this. Get those reps in! 💪

Reply:
Thank you! Is 11 to 14 what looks a bit too rough?